src/ApiGateways/Mobile.Bff.Shopping/aggregator/Startup.cs

```diff
@@ -100,6 +100,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
 
     public static class ServiceCollectionExtensions
     {
+        [Obsolete]
```

Comment:
Please remove as per comment in https://github.com/dotnet-architecture/eShopOnContainers/pull/1344

Reply:
> Please remove as per comment in #1344

ok @mvelosop